components/NavBar.js

```diff
@@ -73,7 +73,7 @@ export const NavBar = ({color}) => (
         flexDirection={['column', 'row']} 
         justifyContent={['flex-start', 'space-around']} 
         alignItems={['flex-start', 'center']} >
-        <Box style={{zIndex: 1}}>
+        <Box ml={23} style={{zIndex: 1}}>
```

Comment:
Lets revert this and leave the logo aligned to the left edge of that `Flex`. The `Container` already takes care of those margins.